pkg/agent/datapath/localBridge.go

```diff
@@ -257,7 +257,11 @@ func (l *LocalBridge) BridgeInit() {
 	if err := l.initFromLocalRedirectTable(sw); err != nil {
 		log.Fatalf("Failed to init local bridge from local redirect table, error: %v", err)
 	}
+}
+
+func (l *LocalBridge) BridgeExtraInit() {
```

Comment:
Add some comments to indicate this is for CNI related flow init.

Reply:
method name changed to BridgeInitCNI